src/render/WorldRender.cpp

```diff
@@ -472,3 +451,84 @@ void Render::debugDrawPath(const World::Waynet::WaynetInstance& waynet, const st
 
 	ddPop();
 }
+
+void ::Render::drawPfx(World::WorldInstance& world, Components::PfxComponent& pfx, const Render::RenderConfig& config)
+{
+	// TODO: Could optimize this into a global vertexbuffer
+
+	if(!bgfx::isValid(pfx.m_ParticleVB))
+		return;
+
+	// Cache
+	static std::vector<Meshes::WorldStaticMeshVertex> quadVertices;
+
+
+	Math::float3 right = config.state.cameraWorld.Rotate(Math::float3(1,0,0)).normalize() * -0.5f; // 0.5 because they get extended into both directions. We want size 1 in total.
+	Math::float3 up = config.state.cameraWorld.Rotate(Math::float3(0,1,0)).normalize() * 0.5f;
+
+	quadVertices.resize(pfx.m_Particles.size() * 6);
+
+	Meshes::WorldStaticMeshVertex test[6];
```

Comment:
Yeah I meant test arr, just spotted it wasn't used anywhere other than in the memset line below